sdk/runanywhere-python/tests/test_download.py

```
@@ -0,0 +1,197 @@
+    with open(local, "wb") as f:
+        f.write(b"x")
+    assert is_remote_source(local) is False
+
+
+def test_assert_remote_supported():
+    # Remote embedder/stt/tts sources are rejected.
+    for kind in ("embedder", "stt", "tts"):
+        with pytest.raises(SDKException):
+            assert_remote_supported("https://x/y.gguf", kind)
+    # llm/vlm remote sources are allowed (no raise).
+    assert_remote_supported("https://x/y.gguf", "llm") is None
```

Ruff (0.15.21)

[warning] Pointless comparison. Did you mean to assign a value? Otherwise, prepend `assert` or remove it.

(B015)